### PR TITLE
Add wait ms

### PR DESCRIPTION
**Pull request by damosvil**

### CODE REVIEW

**Comment by damosvil** [comment]
Added WaitMs puml

**Comment by stylesuxx** [comment]
@damosvil are you going to add the markdown file too, or you want me to do it?

If PRs are not in draft mode I always think it is ready to be merged, that's why I am asking. If there is more coming, feel free to convert the PR into a draft.

**Comment by damosvil** [comment]
Ok, I din't know. I am going to mark it as a draft, to later add the markdown file

File: _pages/WaitMs.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: WaitMs routine
+permalink: /wait_ms/
```
Comment (stylesuxx) [inline _pages/WaitMs.md]: I think we should be consistent with the URLs and seperate with hyphen instead of underscore

**Review by stylesuxx — CHANGES_REQUESTED** [review]

File: _pages/WaitMs.md (hunk 1 above)
Comment (damosvil) [reply _pages/WaitMs.md]: fixed

**Review by stylesuxx — APPROVED** [review]

**Pull request merged**